### PR TITLE
lesson-1-2

### PR DESCRIPTION
**Pull request by leipei92**

### CODE REVIEW

**Comment by JoHeyJo** [comment]
Good job separating your files(modules). Let's make sure we constantly format our files. It will keep code neat and it'll be easier to read. Do a little research as to how you can accomplish this with your IDE. If already installed you can trigger it in the settings or there is a friendly hotkey combo that'll trigger this. If it's not installed you can look into an extension.

File: src/AddTodoForm.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+  const AddTodoForm = () => {
+    return (
+        <form>
+            <label htmlFor="todoTitle"> Title </label>
+            <input id="todoTitle"></input>
```
Comment (JoHeyJo) [inline src/AddTodoForm.js]: What type of input is this? Is there a property that could clearly define it?

File: src/AddTodoForm.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+            <button> Add </button>
```
Comment (JoHeyJo) [inline src/AddTodoForm.js]: What type of button is this? Is there a property that can clearly define it?

File: src/AddTodoForm.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        </form>
+
+    )
+
+  }
+
```
Comment (JoHeyJo) [inline src/AddTodoForm.js:14]: There is a hint in the instructions as to what "type" of element they are.

File: src/AddTodoForm.js (hunk 2 above)
Comment (leipei92) [reply src/AddTodoForm.js]: got it! done

File: src/AddTodoForm.js (hunk 1 above)
Comment (leipei92) [reply src/AddTodoForm.js]: done:)

**Pull request merged**